### PR TITLE
Conversation component now uses component -> store -> connection pattern

### PR DESCRIPTION
**Pull request by philrenaud**
Back to functional again!
Note:
- still using clean conversations endpoint, so if you don't have it enabled, you may get a console error / default message.body insetad
CleanShot 2021-08-20 at 13 27 37

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/conversation/src/index.html [hunk 1]
```diff
@@ -13,10 +13,17 @@
...
+      padding: 0;
+      margin: 0;
+    }
+  </style>
+
   <body>
     <nylas-conversation
       id="demo-conversation"
-      thread_id="4z2ztp19z1p5jj3ofadv18vgw"
+      thread_id="ak2q46em0srqkh7vri8f8vuqj"
```
Comment (philrenaud) [inline components/conversation/src/index.html:26]: Now matching the same thread as `<nylas-email>`

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "build-storybook": "build-storybook -o ./storybook-static --quiet",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "dev": "lerna run --parallel dev",
+    "dev": "lerna run --parallel dev --concurrency 7",
```
Comment (philrenaud) [inline package.json:12]: Lerna internal global flag: https://github.com/lerna/lerna/tree/main/core/global-options#--concurrency

File: components/conversation/src/Conversation.svelte [hunk 3]
```diff
@@ -522,17 +476,6 @@
                     {:else}
                       <p class="snippet">{message.snippet}</p>
                     {/if}
-                    {#if !feedbackSubmittedMessages.includes(message.id)}
```
Comment (philrenaud) [inline components/conversation/src/Conversation.svelte:525]: Decided to nix the Clean Conversation feedback button; it was originally for internal testing only.

**Review by wonka-lighthouse — COMMENTED** [review]
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/agenda/src/index.html |

     
| Device | URL |
|--|--|
| mobile | http://localhost:8000/components/composer/src/index.html |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by ozsivanov — APPROVED** [review]
Nice 👍 , looks like you're missing an import for the `Thread` typing though

File: commons/src/store/conversations.ts [hunk 4]
```diff
@@ -1,14 +1,32 @@
...
   Conversation,
   StoredConversation,
+  ConversationQuery,
   StoredMessage,
 } from "@commons/types/Nylas";
 
 function initializeConversations() {
   const { subscribe, set, update } = writable<Record<string, Conversation>>({});
+
+  let threadsMap: Record<string, Thread> = {};
```
Comment (pooja169usp) [inline commons/src/store/conversations.ts:13]: Looks like we're missing a type import here?

**Review by pooja169usp — APPROVED** [review]

**Review by arjunkalburgi — APPROVED** [review]

**Pull request merged**